src/graphics/text.rs

```diff
@@ -271,41 +274,49 @@ impl Drawable for FontAtlas {
     }
 }
 
+const DEFAULT_TEXT_BUFFER_SIZE: usize = 64;
```

Comment:
Any particular reason for 64 here instead of using the input strings length to initialize the sprite batch buffer?

Reply:
Yes; it should not be a requirement to initialize it with a string, imo. Please note the added `from_cached` method which does not take an init string.

Reply:
I saw it as more of a convenience to allow the user to set the text while making a new text object at the same time, but that's I guess it doesn't make much of a difference either way. Is there any way for the user to specify how large the text buffer should be themselves? Seems wasteful to allocate 64 chars if they don't need 64.

Reply:
64 chars is tiny, we could have a `with_capacity` method similar to the spritebatch, and then a `new` which allocates the default. We could also allocate 0 by default, but that would mean the first call during a draw would allocate a buffer on the GPU for sure. Plus the text is almost always going to change per-frame.